ajax/fuzzysearch.php

```diff
@@ -44,5 +47,5 @@
 try {
     echo json_encode(Html::getMenuFuzzySearchList(), JSON_THROW_ON_ERROR);
 } catch (JsonException $e) {
-    die(500);
+    throw new HttpException(500, 'Error encoding JSON.', $e);
 }
```

Comment:
You should just remove the `try/catch` block. It was present as exceptions were not handled correctly until recently.